Description: accept older major version of crate serial_test
Author: Jonas Smedegaard <dr@jones.dk>
Bug-Debian: https://bugs.debian.org/1093802
Forwarded: not-needed
Last-Update: 2025-08-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -126,7 +126,7 @@
 filetime = "0.2"
 itertools = ">= 0.12, <= 0.13"
 predicates = "3.1.0"
-serial_test = "3.1"
+serial_test = ">= 2.0.0, <= 3"
 temp-env = "0.3.6"
 test-case = "3.3.1"
 thirtyfour_sync = "0.27"
